porcupine/plugins/jump_to_definition.py

```diff
@@ -116,3 +117,6 @@ def generate_jump_request(tab: tabs.FileTab) -> None:
 def setup() -> None:
     get_tab_manager().add_filetab_callback(on_new_filetab)
     menubar.add_filetab_command("Edit/Jump to definition", generate_jump_request)
+    rightclick_menu.add_rightclick_option(
+        "Jump to definition", generate_jump_request, needs_selected_text=True
```

Comment:
This does not need you to select the entire thing:

```suggestion
        "Jump to definition", generate_jump_request
```